.coveragerc

```diff
@@ -28,6 +28,9 @@ omit =
     homeassistant/components/envisalink.py
     homeassistant/components/*/envisalink.py
 
+    homeassistant/components/google.py
```

Comment:
@balloob how would I go about disabling coverage for the google api stuff? Since I'm not failing for those.